src/Exceptionless.Core/Pipeline/030_CheckForRegressionAction.cs

```diff
@@ -23,7 +23,7 @@ public CheckForRegressionAction(IStackRepository stackRepository, SemanticVersio
     public override async Task ProcessBatchAsync(ICollection<EventContext> contexts)
     {
         var stacks = contexts
-            .Where(c => c.Stack is not null && c.Stack.Status != StackStatus.Regressed && c.Stack.DateFixed.HasValue)
+            .Where(c => c.Stack is { Status: StackStatus.Fixed, DateFixed: not null })
```

Comment:
No other status should trigger a regression, so we can save a little bit of work here.